### PR TITLE
Fix 21292: Link directly to https s3 downloads

### PR DESCRIPTION
**Pull request by Geod24**
Companion PR: dlang/downloads.dlang.org#6

### CODE REVIEW

**Review by wilzbach — APPROVED** [review]
I would really prefer if we moved to releases to something that's easier to work with, but I guess this is good enough and eases the pains for the time being.

**Comment by Geod24** [comment]
@wilzbach : So do I, but the issue is orthogonal. We currently have users that click on a download link, and get absolutely nothing. Not even a message from Chrome saying "This file couldn't be downloaded because...". It's embarrassing and needs to be fixed yesterday.

**Comment by Geod24** [comment]
This didn't work, because the files are stored by year internally, not by release...

File: download.dd [hunk 1]
```diff
@@ -238,9 +238,9 @@ Macros:
     SBTN=$(SPANC sig_btn,$(BTN $1,$+)<br>$(BTN $1.sig,sig))
     BTN=<a href="$1" class="btn">$+</a>
     H3I=<h3 class="download">$0</h3>
-    DLSITE=http://downloads.dlang.org/releases/2.x/$(DMDV2)/$0
-    B_DLSITE=http://downloads.dlang.org/pre-releases/2.x/$(B_DMDV2)/$0
-    N_DLSITE=http://downloads.dlang.org/nightlies/dmd-master/$0
+    DLSITE=https://s3.us-west-2.amazonaws.com/downloads.dlang.org/releases/2.x/$(DMDV2)/$0
+    B_DLSITE=https://s3.us-west-2.amazonaws.com/downloads.dlang.org/pre-releases/2.x/$(B_DMDV2)/$0
+    N_DLSITE=https://s3.us-west-2.amazonaws.com/downloads.dlang.org/dmd-master/2.x/$0
```
Comment (Geod24) [inline download.dd:243]: Copy pasta error

**Pull request merged**